plugins/amdgpu/Makefile

```diff
@@ -0,0 +1,14 @@
+all: dummy_plugin.so
+
+dummy_plugin.so: dummy_plugin.c
+	gcc -g -Werror -D _GNU_SOURCE -Wall -shared -nostartfiles dummy_plugin.c -o dummy_plugin.so -iquote ../../../criu/include -iquote ../../criu/include -fPIC
```

Comment:
You should not hardcode CFLAGS but take them from CRIU's build environment. Not sure how to do that correctly, but it needs to be integrated in the existing Makefile infrastructure.

Reply:
Actually a future patch in the series 2 updates the Makefile quite a lot. Could you please take a look at https://github.com/RadeonOpenCompute/criu/blob/amdgpu_rfc-210715-2/plugins/amdgpu/Makefile  and advise